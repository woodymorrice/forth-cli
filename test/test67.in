."pi" 3.14 constant pi . . 3.14 ."yum" set . . .