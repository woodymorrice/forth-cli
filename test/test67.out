3.1400
stack empty
yum
3.1400
stack empty
